app/views/teacher_mailer/deny_email.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>
+  <%= sanitize @reason %>
+</p>
```

Comment:
The only thing is we need to refactor this so it works with the email template PR.
https://github.com/tommywei110/BJC-Teacher-Tracker/pull/28